components/alert/js-alerts/js-alerts.css

```diff
@@ -0,0 +1,142 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+
+.et_js-alerts-container{
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin: 4rem 0rem;
+}
+
+/* js-dismissible alerts start here */
+.dismissible-container{
+    align-items: center;
+    background-color: var(--white-color);
+    border: 1px solid var(--light-gray);
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    margin: 2rem 0rem;
+    padding: 0rem 0rem 2rem 0rem;
+    width: 90%;
+}
+
+.dismissible-container h3{
+    color: var(--secondary-color);
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+.alert{
+    border: 1px solid var(--light-gray);
+    color: var(--light-gray);
+    display: flex;
+    justify-content: space-between;
+    margin: 1rem 0rem;
+    padding: 1rem;
+    width: 90%;
+}
+
+.et_dismissible-simple-primary-alert{
+    background-color: var(--primary-color);
+}
+
+.et_dismissible-simple-secondary-alert{
+    background-color: var(--lightest-blue);
+}
+
+.et_dismissible-success-secondary-alert{
+    background-color: var(--lightest-green);
+}
+
+.et_dismissible-danger-secondary-alert{
+    background-color: var(--lightest-red);
+}
+
+.et_dismissible-warning-secondary-alert{
+    background-color: var(--lightest-yellow);
+}
+
+.et_dismissible-info-secondary-alert{
```

Comment:
Good naming convention

Reply:
okay will look into it!! and will make the required changes, Thankyou so much for the review!!